test-tools/wamr-ide/README.md

```diff
@@ -68,7 +72,15 @@ docker load --input ./wasm-toolchain.tar
 docker load --input ./wasm-debug-server.tar
 ```
 
-##### 3.2 Build docker images on host
+##### 2.1.2 Download the VS Code extension installation file from the GitHub release
+
+From now on, for each GitHub release, we have the same version tagged zip/tar.gz file(e.g. decompress `wamr_ide-1.1.2.tar.gz` `wamr_ide-1.1.2.zip`), which contains .vsix VS Code extension installation file. You can find and easily download in the GitHub release.
+
+#### 2.2 Build WAMR VS Code extension locally(Alternative approach)
+
+You could also build the VS Code extension locally, the following instruction provides a thorough tutorial. It's worth noting that in the local build tutorial we use tag version 1.0 other than the semantic version of WAMR.
```

Comment:
Do we use tag version 1.0 currently? I remember we have changed to 1.1.2?

Reply:
If the user decides to build locally, the default version is 1.0 inside package.json(unlike inside the CI, we change it dynamically based on the release version), and the local build docker image is also tagged with version 1.0 so that they are consistent.